geomesa-lambda/geomesa-lambda-datastore/src/main/scala/org/locationtech/geomesa/lambda/data/LambdaFeatureWriter.scala

```diff
@@ -63,12 +62,10 @@ object LambdaFeatureWriter {
     override def close(): Unit = features.close()
   }
 
-  trait RequiredVisibilityWriter extends AppendLambdaFeatureWriter {
+  trait RequiredVisibilityWriter extends AppendLambdaFeatureWriter with VisibilityChecker {
```

Comment:
Hmm.... I was trying to avoid the repeated trait which extends the specific writers...  I may poke at this for a minute.

Reply:
yeah, i just don't really like exposing the simple feature vars into a trait... seems like a pretty low-level detail. if the feature writer api methods took a feature as input, then we could just mix it in at that level, but since they use the `hasNext`, `next`, `write` api calls it doesn't work very cleanly.